pyatlan/generator/templates/methods/asset/a_p_i_path.jinja2

```diff
@@ -0,0 +1,11 @@
+
+    @classmethod
+    # @validate_arguments()
+    def create(cls, *, apiPathRawURI: str, apiSpecQualifiedName: str) -> {{ entity_def.name }}:
```

Comment:
Need to go from `lowerCamelCase` to `snake_case`, though 🙏

Reply:
Thanks! I'd keep that in mind for future as well 👍